pkg/generators/forwarding/fluentd/generators.go

```diff
@@ -43,9 +43,9 @@ func (engine *ConfigGenerator) Generate(forwarding *logforward.ForwardingSpec) (
 	var outputLabels []string
 	var err error
 
-	logTypes := gatherLogSourceTypes(forwarding.Pipelines)
+	logTypes, appNs := gatherLogSourceTypes(forwarding.Pipelines)
```

Comment:
Both results need to be present and passed down the callstack, thus capturing this in a simple `LogTypeFilter` struct is maybe better. WDYT?

Reply:
will update when merging with GA API implementation.